### DISCUSSION TITLE
ETag missing from Cloudfront.getDistributionConfig when running on client

### DISCUSSION THREAD

**Discussion by FalekPRO** [aws/aws-sdk-js#4086] [General]
I have a test script which runs the method CloudFront.getDistributionConfig:

`CloudFront.getDistributionConfig(id, function (err, data) { })`

But on the response data, the ETag entry is missing, only the DistributionConfig entry is returned.
If I run this code on Node, then the ETag entry is correctly returned, but when running on the client (on browser) it doesn't.

It's necessary a configuration on the Distribution to allow returning ETag to client that I'm not aware of?

Thanks!

**Comment by ajredniwja** [comment]
@FalekPRO Can you please share the code that you are using?

**Comment by FalekPRO** [comment]
> @FalekPRO Can you please share the code that you are using? 


`
var params = {
  Id: 'STRING_VALUE' /* required */
};

cloudfront.getDistributionConfig(params, function(err, data) {
  if (err) console.log(err, err.stack); // an error occurred
  else     console.log(data);           //successful response
});`

**Comment by ajredniwja** [comment]
@FalekPRO Apologies lost track of this issue, is it still a persisting issue?
